lib/jsdom/level2/html.js

```diff
@@ -2048,6 +2048,17 @@ function loadFrame (frame) {
   }
 }
 
+function refreshAccessors(document) {
+  var window = document.defaultView;
+  var frames = Array.prototype.slice.call(document.querySelectorAll("iframe,frame"));
+  frames.forEach(function (frame, i) {
+    delete window[i];
+    defineGetter(window, i, function () {
```

Comment:
Hmm is there a reason not to do `window[i] = frame.contentWindow`?

I guess if you navigated the frame then that would be incorrect, so getter is better. We should add a test for that case!